src/connectObservable.ts

```diff
@@ -13,30 +12,19 @@ import { ConnectorOptions, defaultConnectorOptions } from "./options"
  * observable.
  *
  * @param observable Source observable to be used by the hook.
- * @param options ConnectorOptions:
- *  - unsubscribeGraceTime (= 200): Amount of time in ms that the shared
- *    observable should wait before unsubscribing from the source observable
- *    when there are no new subscribers.
- *  - compare (= Object.is): Equality function.
+ * @param unsubscribeGraceTime (= 200): Amount of time in ms that the shared
+ *        observable should wait before unsubscribing from the source observable
+ *        when there are no new subscribers.
  */
 export function connectObservable<T>(
   observable: Observable<T>,
-  options?: ConnectorOptions<T>,
+  unsubscribeGraceTime = 200,
 ) {
-  const _options = {
-    ...defaultConnectorOptions,
-    ...options,
-  }
-  const sharedObservable$ = distinctShareReplay(_options.compare)(
-    concat(observable, NEVER),
-  )
-
+  const sharedObservable$ = shareLatest<T>(false)(concat(observable, NEVER))
```

Comment:
with `shareLatest(false)` concat NEVER is redundant, right?

Reply:
yes, indeed! thanks!